explainability-core/src/test/java/org/kie/trustyai/explainability/local/tssaliency/TSSaliencyModel.java

```diff
@@ -87,19 +75,18 @@ public double compute(List<Feature> features) {
         double sum = 0.0;
         int randomIndex = 0;
         for (Feature feature : features) { // iterate through features vectpr
-            // System.out.println("feature = " + feature);
 
-            assert feature.getType() == Type.VECTOR; 
+            assert feature.getType() == Type.VECTOR;
```

Comment:
I don't think we have assertion enabled so this is mainly equivalent to a comment